challenges/functions.js

```diff
@@ -27,6 +42,7 @@
 
 // Explanation: 
 
+// In the example below, myFunction is the parent function which is block scoped and function scoped at the same time. The function nestedFunction is the child function of myFunction. The direction of closures goes from inside out, ie. the deepest nested function out to global (ex: grandchild function --> child function --> parent function --> global). Following closure rules, nestedFunction has access to the elements inside its function, and elements outside of the function. Therefore, nestedFunction can access the variable internal because it lies outside of nestedFunction in myFunction. However, myFunction cannot access anything inside nestedFunction because it goes against the closure direction.  
```

Comment:
good answer! try to break it onto different lines -- remember no horizontal scroll bar!